package.json

```diff
@@ -23,16 +23,20 @@
     "prettier-write": "prettier --ignore-unknown --loglevel warn --write .",
     "start:mui": "yarn workspace @okta/odyssey-react-mui dev",
     "start:storybook": "yarn workspace @okta/odyssey-storybook start",
-    "start": "yarn install && npm-run-all --sequential build --parallel dev:source start:storybook",
+    "start": "yarn install && yarn build && yarn concurrently npm:dev:source npm:start:storybook",
```

Comment:
Need the `npm:` prefixes here?

Reply:
With `concurrently`, it needs the `npm:` prefix when you're defining a script name from `package.json`. `npm-run-all` only worked with script names; not custom commands.